beacon_node/lighthouse_network/gossipsub/src/behaviour/tests.rs

```diff
@@ -284,6 +285,7 @@ where
         let fake_endpoint = ConnectedPoint::Dialer {
             address: Multiaddr::empty(),
             role_override: Endpoint::Dialer,
+            port_use: PortUse::New,
```

Comment:
If this is `SO_REUSEADDR` I think we'll want `PortUse::Reuse` everywhere. I don't see any harm in always reusing.

Reply:
yup you are right Michael, updated!